liquid-prep-app/src/app/components/header-title/header-title.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="header-title" style="overflow-x:hidden;" fxFlexFill cdkScrollable>
+    <mat-toolbar class="header-title-header mat-elevation-z4 custom-toolbar" >
+      <mat-toolbar-row style="padding: 0 5px;">
+        <button mat-icon-button class="toolbar-title" aria-label="Icon-button with close icon" (click)="backClicked">
```

Comment:
In the Farm Dashboard, it's a hamburger menu icon that shouldn't execute location.back()
So it's better to let the parent component implement the click callback method